static/js/controllers/inbox.js

```diff
@@ -653,6 +653,19 @@ var feedback = require('../modules/feedback'),
         RecurringProcessManager.stopUpdateRelativeDate();
       });
 
+      Changes({
+        key: 'inbox-user-context',
+        filter: function (change) {
+          var userCtx = Session.userCtx();
```

Comment:
I don't know if calling this is expensive or not (is reading a cookie a form of IO? Who knoooowwwwwwws), but it's worth considering that this will get run over every change (which for admins is every change in the db). Perhaps consider making the name in the userctx (which definitely can't change in the session) static to `inbox.js`?

Reply:
Done